db/migrate/20160906153336_add_delete_projects_endpoint.rb

```diff
@@ -0,0 +1,23 @@
+class AddDeleteProjectsEndpoint < ActiveRecord::Migration
+  def change
+    execute <<-SQL
+      CREATE OR REPLACE FUNCTION "1".delete_project(_project_id integer) RETURNS void
+          LANGUAGE plpgsql
+          AS $$
+              begin
+                update project_transitions pt set most_recent = false where pt.project_id = _project_id;
```

Comment:
@adrianob we need to check permissions here before make updates.
- load the project inside a project var and check public.is_owner_or_admin(p.user_id)
- when project already have online and received some contributions or can delete only when current_state is draft?